docs/guide/database-queries.md

```diff
@@ -153,6 +169,65 @@ query
 
 This example tells the query that its results should be returned in a group of no more than 20 (`.perPage(20)`) and that we want the second group of results based on the active sort order (`.page(2)`). These builders are used in the Apostrophe REST APIs and can be very helpful when dealing with large amounts of content. They use two other builders internally (`limit` and `skip`), but can be easier to use for common situations.
 
+### Schema field builders
+
+Apostrophe automatically creates query builders for most [schema fields](/guide/content-schema.md) configured on piece and page types. Passing a value into a schema field builder refines the query to fetch documents that have the provided value in the matching field.
+
+For example, we might have a select field on a `product` piece type to identify a particular category:
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    fields: {
+      add: {
+        //...
+        category: {
+          type: 'select',
+          label: 'Product Line',
+          choices: [
+            { value: 'professional', label: 'Professional' },
+            { value: 'hobbyist', label: 'Hobby' },
+            { value: 'athletic', label: 'Athletic' }
+          ]
+        }
+      }
+    }
+  }
+  ```
+  <template v-slot:caption>
+    modules/product-page/views/show.html
+  </template>
+</AposCodeBlock>
+
+We can query products matching a particular category:
+
+```javascript
+query.category('athletic')
+```
+
+The following field types get this query builder treatment:
+
+- `string`
+- `slug`
+- `boolean`
+- `checkboxes`
+- `select`
+- `radio`
+- `integer`
+- `float`
+- `url`
+- `date`
+- `relationship`
+
+`relationship` fields get special treatment with four query builders for each field. If the field name is `_products`, then:
+
+- `._products(product._id)` matches only documents related to the product with the provided `_id`. You can also pass an array of product IDs, in which case documents related to *at least one* of those products will match.
+- `.products(product.slug)` (no leading `_`) is similar, but supports matching by the `slug` property. You can also pass an array of product slugs, in which case only documents related to *at least one* of those products will match.
+- `._productsAnd([ _id1, _id2, _id3...])` matches only documents related to *all* of the specified product IDs (one relation is not enough).
+- `.productsAnd([ slug1, slug2, slug3...])` (no leading  `_`) matches only documents related to *all* of the specified product slugs (one relation is not enough).
+
+`_id` properties are useful since they will never change, but slugs can be more readable (for humans) and parsed from URLs.
```

Comment:
I'd say "when visible in URLs," since both ids and slugs work fine in URLs — the issue is readability.